src/index.ts

```diff
@@ -201,6 +188,16 @@ class ClientApp {
         this.externalContacts = new ExternalContactsApi(apiCfg);
     }
 
+    private lookupEnv = (env: string) => {
+        if (env === 'localhost' && __HOST_APP_DEV_ORIGIN__) {
+            this._customPcOrigin = __HOST_APP_DEV_ORIGIN__;
```

Comment:
This side effect is a bit wonky due to the name `lookupEnv` but it's private and internal so not that big of a deal.  Otherwise, we'd need to complicate the return signature and handle it externally, so, not the end of the world.

Reply:
Lol, this PR is approaching the length of time ago where I'm questioning what past me was thinking 😆 Talk about wonk...

Reply:
Alright, this wonkiness and the string type check for the parsed query param value are fixed in [this commit](https://github.com/MyPureCloud/client-app-sdk/pull/75/commits/697bc803d2e76b8e1823159cd6a9f6ffbd85020d).